### DISCUSSION TITLE
not seeing flux models in inference menu

### DISCUSSION THREAD

**Discussion by mjugger** [LykosAI/StabilityMatrix#1001] [General]
I've downloaded flux and a few flux checkpoints as well, but when I go to use them i don't see them in the inference model dropdown. Is there something that I may have forgotten to setup or is this a known bug?

**Comment by mohnjiles** [comment]
For Flux you'll need to click the cogwheel ⚙️ button next to the model dropdown & select "Change Model Loader". Then typically Flux uses UNet unless you have a GGUF version, with which you should use the GGUF model loader

**Reply by mjugger** [reply]
awesome! that did it. thanks. so one other thing, I see there is an add on called flux guidance but it has no settings other than adding it and turning it on or off. does it some how link to CFG? I was under the impression that CFG was guidance whether if it's for flux or non flux. what does flux guidance setting actually do?

**Reply by mohnjiles** [reply]
the "Use Flux Guidance" addon is intended for the "all-in-one" or "easy to use" Flux models that have the vae & text encoders baked into the model like traditional SD1.5/SDXL models, since that uses the "Default" model loader & thus we aren't sure whether or not to use the Flux CFG Scale. Other model loaders will use Flux Guidance by default.

**Reply by mjugger** [reply]
thank you.
